airbyte-cdk/bulk/toolkits/extract-jdbc/src/main/kotlin/io/airbyte/cdk/check/JdbcCheckQueries.kt

```diff
@@ -30,7 +30,7 @@ const val CHECK_QUERIES_PREFIX = "airbyte.connector.check.jdbc"
 class JdbcCheckQueries {
 
     // Micronaut configuration objects work better with mutable properties.
-    lateinit var queries: List<String>
+    protected var queries: List<String> = emptyList()
```

Comment:
Apparently `protected` is preferred over `private` for making properties non-public with micronaut; because otherwise micronaut has to use reflection at runtime instead of being able to do its thing at compile-time